src/Parser/AfflictionWarlock/Modules/Features/CastEfficiency.js

```diff
@@ -0,0 +1,52 @@
+import SPELLS from 'common/SPELLS';
+
+import CoreCastEfficiency from 'Parser/Core/Modules/CastEfficiency';
+
+const SPELL_CATEGORY = {
+    ROTATIONAL: 'Rotational Spell',
+    COOLDOWNS: 'Cooldown',
+};
+
+class CastEfficiency extends CoreCastEfficiency {
+  static CPM_ABILITIES = [
```

Comment:
Might re-iterate on that. Initially I had a lot of spells there but I thought it just cluttered the screen with unimportant info (and suggestions, like "you casted 0/2 Unending Resolves" (a defensive cooldown)), but that was before I knew of noSuggestion: true. I'll have a look at that.